LanguageFeatures/Augmentation-libraries/augmented_expression_A10_t02_lib.dart

```diff
@@ -362,3 +362,84 @@ augment extension Ext {
 // [cfe] unspecified
   }
 }
+
+augment extension type ET {
+  static var staticVariable = () {
+    int augmented = 42;
+//      ^^^^^^^^^
+// [analyzer] unspecified
+// [cfe] unspecified
+    for (int augmented = 0; augmented < 0; augmented++) {}
```

Comment:
Maybe avoid using the declaration? We could have a bug whereby the declaration is accepted, but the usages of `augmented` as an expression give rise to an error, and then we'd have a false positive on this error. Perhaps the following would suffice:
```suggestion
    for (int augmented = 0;;) {}
```

Reply:
(I guess this question wasn't raised when the non-new text was created, e.g., line 359)